js/palette.js

```diff
@@ -1,85 +1,85 @@
 //NES color palette
 let palette_color_array = [
-    "#fcfcfc",
-    "#f8f8f8",
-    "#bcbcbc",
-    "#7c7c7c",
-    
-    "#a4e4fc",
-    "#3cbcfc",
-    "#0078f8",
-    "#0007fc",
-    
-    "#b8b8f8",
-    "#6888fc",
-    "#0059f8",
-    "#0004bc",
-    
-    "#d8b8f8",
-    "#9878f8",
-    "#6846fc",
-    "#432abc",
+	"#fcfcfc",
+	"#f8f8f8",
+	"#bcbcbc",
+	"#7c7c7c",
+	
+	"#a4e4fc",
+	"#3cbcfc",
+	"#0078f8",
+	"#0007fc",
+	
+	"#b8b8f8",
+	"#6888fc",
+	"#0059f8",
+	"#0004bc",
+	
+	"#d8b8f8",
+	"#9878f8",
+	"#6846fc",
+	"#432abc",
 
-    "#f8b8f8",
-    "#f878f8",
-    "#d801cc",
-    "#940084",
+	"#f8b8f8",
+	"#f878f8",
+	"#d801cc",
+	"#940084",
 
-    "#f8a4c0",
-    "#f878f8",
-    "#d801cc",
-    "#a80020",
-    
-    "#f0cfb0",
-    "#f87758",
-    "#f83701",
-    "#a80e00",
-    
-    "#fce0a8",
-    "#fc9f44",
-    "#e45b11",
-    "#881400",
+	"#f8a4c0",
+	"#f878f8",
+	"#d801cc",
+	"#a80020",
+	
+	"#f0cfb0",
+	"#f87758",
+	"#f83701",
+	"#a80e00",
+	
+	"#fce0a8",
+	"#fc9f44",
+	"#e45b11",
+	"#881400",
 
-    "#f8d878",
-    "#f8b801",
-    "#ac7b01",
-    "#503000",
+	"#f8d878",
+	"#f8b801",
+	"#ac7b01",
+	"#503000",
 
-    "#d8f878",
-    "#b9f819",
-    "#02b801",
-    "#017800",
+	"#d8f878",
+	"#b9f819",
+	"#02b801",
+	"#017800",
 
-    "#b8f8b8",
-    "#58d854",
-    "#01a801",
-    "#016800",
+	"#b8f8b8",
+	"#58d854",
+	"#01a801",
+	"#016800",
 
-    "#b8f8d8",
-    "#58f898",
-    "#01a844",
-    "#015800",
+	"#b8f8d8",
+	"#58f898",
+	"#01a844",
+	"#015800",
 
-    "#00fcfc",
-    "#00e8d8",
-    "#008888",
-    "#004058",
+	"#00fcfc",
+	"#00e8d8",
+	"#008888",
+	"#004058",
 
-    "#c4c4c4",
-    "#787878",
-    "#000001",
-    "#000000",
+	"#c4c4c4",
+	"#787878",
+	"#000001",
+	"#000000",
 ];
 
 //Classic Gameboy and Gameboy pocket color palettes
 let gb_palette_color_array = [
-    "#9bbc0f",
-    "#8bac0f",
-    "#306230",
-    "#0f380f",
+	"#9bbc0f",
+	"#8bac0f",
+	"#306230",
+	"#0f380f",
 
-    "#ffffff",
-    "#a9a9a9",
-    "#545454",
-    "#000000",
+	"#ffffff",
+	"#a9a9a9",
+	"#545454",
+	"#000000",
 ];
```

Comment:
Should there be an empty line at the end of the file here?

If there should be and you want to add, that would be great. If there was no empty line before, no worries. 👍